Makefile

```diff
@@ -0,0 +1,13 @@
+CONTAINER_MANAGER ?= podman
+
+IMG ?= quay.io/redhat-developer/openshift-local-extension:latest
+IMG_CACHE ?= quay.io/rhqp/openshift-local-extension-cache:latest
+
+# Build the container cache image
+.PHONY: build-cache
+build-cache: 
+	${CONTAINER_MANAGER} build -t ${IMG_CACHE} -f oci/Containerfile.cache .
```

Comment:
Who maintains the lifecycle of this cache?

Reply:
the idea would be self managed on every merge to main a gh action will build the image and push it